File: Diploma/LPU-4th-sem/Java/src/com/nitin/AreaOfCircle.java
package com.nitin;

import java.util.Scanner;

public class AreaOfCircle {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);

        System.out.println("Enter the radius: ");
        int radius = in.nextInt();
        double pi = 3.142, area;
        area = pi * radius * radius;
        System.out.println("Area of circle is :" + area);
    }
}

File: B tech/CPP/OOPS/areaofCircle.cpp
#include <iostream>
using namespace std;

class Circle {
    float r;
    public:
        Circle(float radius) {
            r = radius;
        }
        friend void show_data(Circle);
};

void show_data(Circle c) {
    const float pi = 3.14;
    cout << "Area of circle is: " << pi*c.r*c.r << endl;
}

int main() {
    Circle c1(10);
    show_data(c1);
    return 0;
}